### PR TITLE
Define ClockFormatTrait

### PR DESCRIPTION
**Pull request by alexhqwang**
Add a trait to set the clock display format.
Options are 12-hour, 24-hour, and locale default.

### CODE REVIEW

File: weave/trait/time/clock_format_trait.proto [hunk 1]
```diff
@@ -0,0 +1,28 @@
+// This file specifies a Weave Common trait for setting the clock format.
```
Comment (gerickson) [inline weave/trait/time/clock_format_trait.proto]: Please include a standard header per other traits in this repo.

**Review by gerickson — CHANGES_REQUESTED** [review]
See comments regarding header.

File: weave/trait/time/clock_format_trait.proto (hunk 1 above)
Comment (alexhqwang) [reply weave/trait/time/clock_format_trait.proto]: Done, not sure if there was a way to amend the first commit like in gerrit (force push?) so I just pushed another commit per the github instructions.

**Review by gerickson — APPROVED** [review]

**Review by gerickson — APPROVED** [review]

**Review by gerickson — APPROVED** [review]

**Comment by gerickson** [comment]
@mrjerryjohns, good to merge? The PR has passed all checks, there's a hiccup between GitHub and Travis—if you click through the pending integration, it passed.

**Pull request merged**